src/Elasticsearch/Namespaces/IndicesNamespace.php

```diff
@@ -275,6 +275,25 @@ public function snapshotIndex($params = array())
     }
 
     /**
+     * @param $params array Associative array of parameters
```

Comment:
Oh, can we also add the the list of parameters here too?  You can copy the formatting from another of the methods above